### ISSUE TITLE
change CL import from cl-basic to cl-base

### ISSUE THREAD

**Issue by balhoff** [geneontology/go-ontology#19984]
Currently we're missing taxon constraints that should be imported from CL. See #17613

**Comment by balhoff** [comment]
A few unsatisfiable classes are preventing use of cl-base, all connected to 'Malphighian tubule'. I think the problem lies in the logical definition for 'mesenchymal stem cell' in CL (cc @cmungall @dosumis @matentzn):

## [Malpighian tubule morphogenesis](http://purl.obolibrary.org/obo/GO_0007443) EquivalentTo [Nothing](http://www.w3.org/2002/07/owl#Nothing) ##

  - [Malpighian tubule morphogenesis](http://purl.obolibrary.org/obo/GO_0007443) EquivalentTo [anatomical structure morphogenesis](http://purl.obolibrary.org/obo/GO_0009653) and ([results in morphogenesis of](http://purl.obolibrary.org/obo/RO_0002298) some [Malpighian tubule](http://purl.obolibrary.org/obo/UBERON_0001054))
    - [Malpighian tubule morphogenesis](http://purl.obolibrary.org/obo/GO_0007443) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [embryonic hindgut morphogenesis](http://purl.obolibrary.org/obo/GO_0048619)
      - [embryonic hindgut morphogenesis](http://purl.obolibrary.org/obo/GO_0048619) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [hindgut morphogenesis](http://purl.obolibrary.org/obo/GO_0007442)
        - [hindgut morphogenesis](http://purl.obolibrary.org/obo/GO_0007442) EquivalentTo [anatomical structure morphogenesis](http://purl.obolibrary.org/obo/GO_0009653) and ([results in morphogenesis of](http://purl.obolibrary.org/obo/RO_0002298) some [hindgut](http://purl.obolibrary.org/obo/UBERON_0001046))
          - [hindgut](http://purl.obolibrary.org/obo/UBERON_0001046) SubClassOf [develops from](http://purl.obolibrary.org/obo/RO_0002202) some [mesenchyme](http://purl.obolibrary.org/obo/UBERON_0003104)
            - [develops from](http://purl.obolibrary.org/obo/RO_0002202) SubPropertyOf: [has developmental contribution from](http://purl.obolibrary.org/obo/RO_0002254)
            - [mesenchyme](http://purl.obolibrary.org/obo/UBERON_0003104) EquivalentTo [tissue](http://purl.obolibrary.org/obo/UBERON_0000479) and ([composed primarily of](http://purl.obolibrary.org/obo/RO_0002473) some [mesenchymal stem cell](http://purl.obolibrary.org/obo/CL_0000134))
              - [composed primarily of](http://purl.obolibrary.org/obo/RO_0002473) SubPropertyOf: [has_part](http://purl.obolibrary.org/obo/BFO_0000051)
              - [mesenchymal stem cell](http://purl.obolibrary.org/obo/CL_0000134) EquivalentTo [connective tissue cell](http://purl.obolibrary.org/obo/CL_0002320) and ([has plasma membrane part](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001345](http://purl.obolibrary.org/obo/PR_000001345)) and ([has plasma membrane part](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001800](http://purl.obolibrary.org/obo/PR_000001800)) and ([has plasma membrane part](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001843](http://purl.obolibrary.org/obo/PR_000001843)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [osteoblast differentiation](http://purl.obolibrary.org/obo/GO_0001649)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [cell migration](http://purl.obolibrary.org/obo/GO_0016477)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [neuron differentiation](http://purl.obolibrary.org/obo/GO_0030182)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [epithelial cell differentiation](http://purl.obolibrary.org/obo/GO_0030855)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [fat cell differentiation](http://purl.obolibrary.org/obo/GO_0045444)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [cardiac muscle cell differentiation](http://purl.obolibrary.org/obo/GO_0055007)) and ([capable_of](http://purl.obolibrary.org/obo/RO_0002215) some [chondroblast differentiation](http://purl.obolibrary.org/obo/GO_0060591)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/cl#lacks_plasma_membrane_part) some [MHC class II protein complex](http://purl.obolibrary.org/obo/GO_0042613)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/cl#lacks_plasma_membrane_part) some [PR_000001003](http://purl.obolibrary.org/obo/PR_000001003)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/cl#lacks_plasma_membrane_part) some [PR_000001006](http://purl.obolibrary.org/obo/PR_000001006))
                - [cardiac muscle cell differentiation](http://purl.obolibrary.org/obo/GO_0055007) EquivalentTo [cell differentiation](http://purl.obolibrary.org/obo/GO_0030154) and ([results in acquisition of features of](http://purl.obolibrary.org/obo/RO_0002315) some [cardiac muscle cell](http://purl.obolibrary.org/obo/CL_0000746))
                  - [cardiac muscle cell](http://purl.obolibrary.org/obo/CL_0000746) SubClassOf [develops from](http://purl.obolibrary.org/obo/RO_0002202) some [cardiac muscle myoblast](http://purl.obolibrary.org/obo/CL_0000513)
                    - [cardiac muscle myoblast](http://purl.obolibrary.org/obo/CL_0000513) EquivalentTo [myoblast](http://purl.obolibrary.org/obo/CL_0000056) and ([part of](http://purl.obolibrary.org/obo/BFO_0000050) some [cardiac muscle tissue](http://purl.obolibrary.org/obo/UBERON_0001133)) and ([develops into](http://purl.obolibrary.org/obo/RO_0002203) some [cardiac muscle cell](http://purl.obolibrary.org/obo/CL_0000746))
                      - [cardiac muscle tissue](http://purl.obolibrary.org/obo/UBERON_0001133) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [heart](http://purl.obolibrary.org/obo/UBERON_0000948)
                        - [heart](http://purl.obolibrary.org/obo/UBERON_0000948) SubClassOf [structure with developmental contribution from neural crest](http://purl.obolibrary.org/obo/UBERON_0010314)
                          - [structure with developmental contribution from neural crest](http://purl.obolibrary.org/obo/UBERON_0010314) EquivalentTo [anatomical structure](http://purl.obolibrary.org/obo/UBERON_0000061) and ([has developmental contribution from](http://purl.obolibrary.org/obo/RO_0002254) some [neural crest](http://purl.obolibrary.org/obo/UBERON_0002342))
                            - [neural crest](http://purl.obolibrary.org/obo/UBERON_0002342) SubClassOf [only_in_taxon](http://purl.obolibrary.org/obo/RO_0002160) some [Craniata <chordates>](http://purl.obolibrary.org/obo/NCBITaxon_89593)
                              - [Craniata <chordates>](http://purl.obolibrary.org/obo/NCBITaxon_89593) SubClassOf [Chordata](http://purl.obolibrary.org/obo/NCBITaxon_7711)
                                - [Chordata](http://purl.obolibrary.org/obo/NCBITaxon_7711) SubClassOf [Deuterostomia](http://purl.obolibrary.org/obo/NCBITaxon_33511)
                              - [only_in_taxon](http://purl.obolibrary.org/obo/RO_0002160) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)
                  - [results in acquisition of features of](http://purl.obolibrary.org/obo/RO_0002315) SubPropertyOf: [results in developmental progression of](http://purl.obolibrary.org/obo/RO_0002295)
      - [part of](http://purl.obolibrary.org/obo/BFO_0000050) SubPropertyOf: [overlaps](http://purl.obolibrary.org/obo/RO_0002131)
    - [results in morphogenesis of](http://purl.obolibrary.org/obo/RO_0002298) SubPropertyOf: [results in developmental progression of](http://purl.obolibrary.org/obo/RO_0002295)
    - [Malpighian tubule](http://purl.obolibrary.org/obo/UBERON_0001054) SubClassOf [only_in_taxon](http://purl.obolibrary.org/obo/RO_0002160) some [Arthropoda](http://purl.obolibrary.org/obo/NCBITaxon_6656)
      - [Arthropoda](http://purl.obolibrary.org/obo/NCBITaxon_6656) SubClassOf [Panarthropoda](http://purl.obolibrary.org/obo/NCBITaxon_88770)
        - [Panarthropoda](http://purl.obolibrary.org/obo/NCBITaxon_88770) SubClassOf [Ecdysozoa](http://purl.obolibrary.org/obo/NCBITaxon_1206794)
          - [Ecdysozoa](http://purl.obolibrary.org/obo/NCBITaxon_1206794) SubClassOf [Protostomia](http://purl.obolibrary.org/obo/NCBITaxon_33317)
  - [in taxon](http://purl.obolibrary.org/obo/RO_0002162) some [Protostomia](http://purl.obolibrary.org/obo/NCBITaxon_33317) DisjointWith [in taxon](http://purl.obolibrary.org/obo/RO_0002162) some [Deuterostomia](http://purl.obolibrary.org/obo/NCBITaxon_33511)
  - [results in developmental progression of](http://purl.obolibrary.org/obo/RO_0002295) o [in taxon](http://purl.obolibrary.org/obo/RO_0002162) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)
  - [capable_of](http://purl.obolibrary.org/obo/RO_0002215) o [in taxon](http://purl.obolibrary.org/obo/RO_0002162) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)
  - [has_part](http://purl.obolibrary.org/obo/BFO_0000051) o [in taxon](http://purl.obolibrary.org/obo/RO_0002162) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)
  - [overlaps](http://purl.obolibrary.org/obo/RO_0002131) o [in taxon](http://purl.obolibrary.org/obo/RO_0002162) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)
  - [has developmental contribution from](http://purl.obolibrary.org/obo/RO_0002254) o [in taxon](http://purl.obolibrary.org/obo/RO_0002162) SubPropertyOf: [in taxon](http://purl.obolibrary.org/obo/RO_0002162)

**Comment by dosumis** [comment]
Yeh - this should go.  It's just wrong:

> mesenchyme EquivalentTo tissue and (composed primarily of some mesenchymal stem cell)

Can you check whether deleting this axiom is sufficient to remove the inconsistency?

There are also multiple historic tickets on problems with mesenchymal stem cell:

https://github.com/obophenotype/cell-ontology/issues?q=is%3Aissue+is%3Aopen++mesenchymal+stem+cell  I think we should put this on the agenda for CL calls.

**Comment by balhoff** [comment]
@dosumis if I remove both:

- `mesenchyme EquivalentTo tissue and (composed primarily of some mesenchymal stem cell)`
- `mesenchyme SubClassOf composed primarily of some mesenchymal stem cell` (added by the Uberon release process)

the unsatisfiable classes go away. So should I open an issue at Uberon?

**Comment by dosumis** [comment]
> So should I open an issue at Uberon?

Yes please.  I'll try to do the edit tomorrow.

**Comment by cmungall** [comment]
Hold on hold on, surely the problem here is in CL

Maybe the Uberon def is overstated but surely there should be some kind of existential path from mesenchyme to MSC; as far as TCs go it just has to be an existential linkage, e.g through derivation. So even if we change to something like M subClassOf has-part some deevelops-from some MSC we still get the violation.

Meanwhile the CL logical axioms are massively problematic. Look at all those human markers. 

Let's look at the provenance of the CL definition: https://pubmed.ncbi.nlm.nih.gov/17986482/

Not only is this *human* specific, it's specific to *BONE* MSCs. There is terminological confusion about the term MSC. Sometimes it's used as a shorthand for bone MSC but there are also placental MSCs etc

![image](https://user-images.githubusercontent.com/50745/93276873-a2a24800-f775-11ea-9d00-aa376cee163d.png)

The core definition in CL is _A connective tissue cell that normally gives rise to other cells that are organized as three-dimensional masses._ which also sounds fairly species-generic

**Comment by cmungall** [comment]
Oh, @balhoff you already reported this in 2017! https://github.com/obophenotype/cell-ontology/issues/474

**Comment by cmungall** [comment]
I made an uberon release which should fix the immediate problem

we should do a CL release too

**Comment by balhoff** [comment]
Thanks, the Uberon release fixed the unsatisfiabilities. New minor issue: https://github.com/obophenotype/cell-ontology/issues/734

**Comment by balhoff** [comment]
Now blocked by https://github.com/obophenotype/cell-ontology/issues/757

**Comment by balhoff** [comment]
After newest CL release, one more blocker, which I think is a GO issue: #20176
